File: VistaLOAN/VistaLOAN.Web/Imports/ServerTypings/ScriptUserDefinition.ts
namespace VistaLOAN {
    export interface ScriptUserDefinition {
        Username?: string;
        DisplayName?: string;
        IsAdmin?: boolean;
        FundControlInformationId?: number;
        BaseCurrencyId?: number;
        BaseCurrency?: string;
        ZoneID?: number;
        FundControlName?: string;
        ZoneName?: string;
        FinancialYearName?: string;
        EmpId?: number;
        Permissions?: { [key: string]: boolean };
        DesignationName?: string;
        LoanTypeInformationId?: number;
        LoanTypeName?: string;
    }
}

File: VistaLOAN/VistaLOAN.Web/Modules/Administration/User/Authentication/ScriptUserDefinition.cs
namespace VistaLOAN
{
    using Serenity.ComponentModel;
    using System;
    using System.Collections.Generic;

    /// <summary>
    /// This data will be available from script code using a dynamic script.
    /// Add properties you need from script code and set them in UserEndpoint.GetUserData.
    /// </summary>
    [ScriptInclude]
    public class ScriptUserDefinition
    {
        public String Username { get; set; }
        public String DisplayName { get; set; }
        public Boolean IsAdmin { get; set; }
        public int FundControlInformationId { get; set; }
        public int BaseCurrencyId { get; set; }
        public string BaseCurrency { get; set; }
        public int ZoneID { get; set; }
        public string FundControlName { get; set; }
        public string ZoneName { get; set; }
        public string FinancialYearName { get; set; }
        public int EmpId { get; set; }
        public Dictionary<string, bool> Permissions { get; set; }
        public string DesignationName { get; set; }
        public int LoanTypeInformationId { get; set; }
        public string LoanTypeName { get; set; }
    }
}